src/main/java/mate/academy/spring/dao/MovieSessionDao.java

```diff
@@ -9,4 +9,8 @@ public interface MovieSessionDao extends GenericDao<MovieSession> {
     List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
 
     Optional<MovieSession> get(Long id);
+
+    MovieSession update(MovieSession movieSession);
+
+    void delete(Long id);
```

Comment:
```suggestion
    void delete(MovieSession movieSession);
```

it's better to get the object via get in the controller and pass it here